src/odemis/acq/stream/_live.py

```diff
@@ -724,6 +721,17 @@ class BrightfieldStream(CameraStream):
       all the spectrum), or a "light" emitter (with multiple channels, for
       various spectra). To activate the light, it just turns on all the channels.
     """
+    def __init__(self, name, detector, dataflow, emitter, emtvas=None, **kwargs):
+
+        if emitter is not None:
+            # Current power VA representing power for one channel only
+            cp_range = (emitter.power.range[0][0], emitter.power.range[1][0])
```

Comment:
Could a `BrightfieldStream` emitter have something other than range in its power?  In other words should I check if range exists here?
@pieleric